From f921c6221736f12f9bfdb303dfd354b824860765 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Sat, 26 Feb 2022 12:32:12 +0800
Subject: [PATCH 3/6] SystemUI: Use Monet colors for power menu

It's odd that the power menu doesn't use Monet colors, unlike most of
System UI. This makes it feel more integrated with the rest of the
system.

Change-Id: Iaffb178fe2c09a33925cb40e89d78640fab2b23c
---
 packages/SystemUI/res/values-night/colors.xml |  7 +++++++
 packages/SystemUI/res/values/colors.xml       | 10 +++++-----
 2 files changed, 12 insertions(+), 5 deletions(-)

diff --git a/packages/SystemUI/res/values-night/colors.xml b/packages/SystemUI/res/values-night/colors.xml
index b98694e2fae7..3a6b62dfe1c8 100644
--- a/packages/SystemUI/res/values-night/colors.xml
+++ b/packages/SystemUI/res/values-night/colors.xml
@@ -99,4 +99,11 @@
     <color name="accessibility_floating_menu_background">#B3000000</color> <!-- 70% -->
 
     <color name="people_tile_background">@android:color/system_accent2_800</color>
+
+    <!-- Colors for Power Menu Lite -->
+    <color name="global_actions_lite_background">@*android:color/primary_device_default_dark</color>
+    <color name="global_actions_lite_button_background">@*android:color/surface_dark</color>
+    <color name="global_actions_lite_text">@*android:color/foreground_device_default_dark</color>
+    <color name="global_actions_lite_emergency_background">@*android:color/surface_dark</color>
+    <color name="global_actions_lite_emergency_icon">@*android:color/accent_device_default_dark</color>
 </resources>
diff --git a/packages/SystemUI/res/values/colors.xml b/packages/SystemUI/res/values/colors.xml
index f539b0cf68a1..9ea915c1cb3f 100644
--- a/packages/SystemUI/res/values/colors.xml
+++ b/packages/SystemUI/res/values/colors.xml
@@ -53,11 +53,11 @@
     <color name="global_actions_emergency_text">@color/GM2_grey_100</color>
 
     <!-- Colors for Power Menu Lite -->
-    <color name="global_actions_lite_background">#191C18</color>
-    <color name="global_actions_lite_button_background">#303030</color>
-    <color name="global_actions_lite_text">#F0F0F0</color>
-    <color name="global_actions_lite_emergency_background">#F85D4D</color>
-    <color name="global_actions_lite_emergency_icon">@color/GM2_grey_900</color>
+    <color name="global_actions_lite_background">@*android:color/primary_device_default_light</color>
+    <color name="global_actions_lite_button_background">@*android:color/surface_light</color>
+    <color name="global_actions_lite_text">@*android:color/foreground_device_default_light</color>
+    <color name="global_actions_lite_emergency_background">@*android:color/surface_light</color>
+    <color name="global_actions_lite_emergency_icon">@*android:color/accent_device_default_light</color>
 
     <color name="global_actions_shutdown_ui_text">@color/control_primary_text</color>
 
-- 
2.32.0 (Apple Git-132)

